simulator/simulator.go

```diff
@@ -22,7 +23,7 @@ const (
 type Manager interface {
 	LaunchWithGUI(simulatorID string) error
 	ResetLaunchServices() error
-	Boot(id string) error
+	Boot(device destination.Device) error
```

Comment:
This seems like a breaking change. As I saw only the xcode-test and xcode-start-simulator steps use this so will you update both?